ui/js/controllers/perf/compare.js

```diff
@@ -190,15 +226,15 @@ perf.controller('CompareResultsCtrl', [
 
                     if (testName.indexOf("summary") > 0) {
                         var detailsLink = 'perf.html#/comparesubtest?';
-                        detailsLink += _.map(_.pairs({
+                        detailsLink += $httpParamSerializer({
```

Comment:
Do you want to bundle this change into a seperate pull request, filed against bug 1285004? Might as well get this one in sooner.